src/Semantic/UtilDisabled.hs

```diff
@@ -1,406 +0,0 @@
-{-# LANGUAGE CPP, ConstraintKinds, Rank2Types, ScopedTypeVariables, TypeFamilies, TypeOperators #-}
-{-# OPTIONS_GHC -Wno-missing-signatures -Wno-missing-export-lists #-}
-
--- This module is *NOT* included in any build targets It is where we
--- store code that is useful for debugging in GHCi but that takes too
--- much time to build regularly. If you need one of these functions,
--- copy it temporarily to Util or some other appropriate place. Be aware
--- that these functions put tremendous strain on the typechecker.
```

Comment:
We really aren’t going to revisit this.

Reply:
Oops hit “resolve” too early. But I agree. It’s bit-rotted and it’s a holdover from days of alacarte ASTs.